scripts/mkcloudcloud/deploy.sh

```diff
@@ -57,6 +61,15 @@ export want_tempest_proposal=1
 ./mkcloud "\$@"
 EOF
 chmod a+x cloud9
+
+if [ ! -f ~/.ssh/id_rsa.pub ]; then
+  ssh-keygen -t rsa -f /root/.ssh/id_rsa -N ""
```

Comment:
```suggestion
  ssh-keygen -t rsa -f ~/.ssh/id_rsa -N ""
```

Reply:
We have that currently in mkcloud's `function prepare` 
We should move it into shared `scripts/lib/mkcloud-onhost.sh` to have it DRY / a single point of truth

Reply:
I don't agree with this because this admin VM is not deployed with `mkcloud` but with a heat template. I would prefer to keep them separated.

Reply:
@JanZerebecki fixed the ssh directory as you suggested.